libs/platform/kernel/platform_kernel.vcxproj.filters

```diff
@@ -58,7 +58,7 @@
     <ClCompile Include="..\ebpf_error.c">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\ebpf_etw.c">
+    <ClCompile Include="..\ebpf_tracelog.c">
```

Comment:
re-sort in alphabetical order

Reply:
`.vcxproj` files are managed by Visual Studio and are designed to be ordering-agnostic; also, they are not user-visible as source code (in fat, in this specific file all are source files are unordered).